packages/react-components/src/components/VideoGallery/LocalVideoCameraButton.tsx

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { IButtonStyles, IconButton } from '@fluentui/react';
+import React from 'react';
+import { OptionsDevice } from '../DevicesButton';
+
+export interface LocalVideoCameraButtonProps {
+  /** Array of cameras available to the user. */
+  cameras?: OptionsDevice[];
+  /** Currently selected camera in the local video stream. */
+  currentCamera?: OptionsDevice;
+  /** callback function to change video feed. */
+  cycleCamera: (device: OptionsDevice) => Promise<void>;
+}
+/**
+ * local video tile camera cycle button - for use on mobile screens only.
+ * @param props
+ * @returns
+ * @private
+ */
+export const LocalVideoCameraButton = (props: LocalVideoCameraButtonProps): JSX.Element => {
+  const { cameras, currentCamera, cycleCamera } = props;
+  const localCameraSwitcherStyles: IButtonStyles = {
+    root: {
+      position: 'absolute',
+      right: '0',
+      color: 'white'
+    }
+  };
+  return (
+    <IconButton
+      styles={localCameraSwitcherStyles}
+      iconProps={{ iconName: 'CameraSwitch' }}
+      ariaLabel={'Cycle Camera Button'}
+      onClick={() => {
```

Comment:
Use `useMemo` to avoid rerendering `IconButton` when no other props have changed.

Reply:
are you talking about the whole button? or just the on click function?

Reply:
Curious about this comment myself @prprabhu-ms .
I am unable to see a need for using useMemo here.
This component will re-render only when one of the props changes and all those props are being used in IconButton as well.